website/docs/r/attestation.html.markdown

```diff
@@ -0,0 +1,111 @@
+---
+subcategory: "Attestation"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_attestation"
+description: |-
+  Manages a Attestation Resource.
+---
+
+# azurerm_attestation
+
+Manages a Attestation Resource.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "UK South"
+}
+
+resource "azurerm_attestation" "example" {
+  name                = "example-attestationprovider"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+}
+```
+
+## Example Usage with PEM File
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "UK South"
+}
+
+resource "azurerm_attestation" "example" {
+  name                = "example-attestationprovider"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+
+  policy_signing_certificate_data = file("./example/cert.pem")
+}
+```
+
+## Example Usage with PEM Certificate String
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "UK South"
+}
+
+resource "azurerm_attestation" "example" {
+  name                = "example-attestationprovider"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+
+  policy_signing_certificate_data = <<EOT
+-----BEGIN CERTIFICATE-----
+MIIBzDCCAS2gAwIBAgIBATAKBggqhkjOPQQDBDAQMQ4wDAYDVQQKEwVFTkNPTTAe
+...
+lXOCt2Yf4dsFsijuvzbJBdx75d4Ef5TRHPcg+PHNZgg=
+-----END CERTIFICATE-----
+EOT
+}
+```
```

Comment:
I think we can remove this example, since the file one should be the most common?

Reply:
Fixed